### PR TITLE
GitHubSync update

### PR DESCRIPTION
**Pull request by particularbot**
This is an automated synchronization PR.

The following source template repositories were used:
* particular/RepoStandards

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -3,14 +3,8 @@ If you do not agree to these terms, do not access the NServiceBus code.
...
 Your license to the NServiceBus source and/or binaries is governed by the Reciprocal Public License 1.5 (RPL1.5) license as described here: 
 
-http://www.opensource.org/licenses/rpl1.5.txt
+https://www.opensource.org/licenses/rpl1.5.txt
 
 If you do not wish to release the source of software you build using NServiceBus, you may use NServiceBus source and/or binaries under the License Agreement described here:
 
-http://particular.net/LicenseAgreement
-
-### [OWin/AspNetKatana](https://github.com/aspnet/AspNetKatana/)
```
Comment (ramonsmits) [inline LICENSE.md:12]: Interesting, for now I think we should just exclude the license, but do update the links to http.

Maybe add a feature for reposync to compare the start?

Comment (andreasohlund) [reply LICENSE.md:12]: Isn't the license meant to convey our license? => and we should move the list if things that we use to a separate file?

Comment (ramonsmits) [reply LICENSE.md:12]: @andreasohlund Unsure, AFAIK all these licenses should be listed in the deployment package for all dependencies in an installer.

Comment (ramonsmits) [reply LICENSE.md:12]: Maybe this can simply be fixed by having a LICENSE.md file in each respective folder?

Comment (ramonsmits) [reply LICENSE.md:12]: I searched for various solutions:

1. Have a COPYING file in root, pointing to all LICENSE/COPYING files
   * https://github.com/marcelhollerbach/efl-meson-ci/blob/ce2e1a937b2ee14bdfc73b4aa999c00230cef4d8/COPYING
2. one file with licenses

   a. Add a `3rd-party-licenses.,txt` file: https://github.com/UniMelb-source/vivo/blob/1d69d71be4875f53238f5c8a156170a2570522d7/doc/3rd-party-licenses.txt

   b. Have a `NOTICE` file mentioning all other licenses : https://github.com/UniteIdeas/linksSDGs-Tagger/blob/master/solr-5.4.0/NOTICE.txt

   c. Have an `ACKNOWLEDGEMENT.txt` : https://github.com/wuwei-spring/broadxt/blob/master/third_party/ACKNOWLEDGEMENT.txt

   d. Have a `CREDITS.txt` : https://github.com/dealnewscentral/dncentral/blob/master/CREDITS.txt

3. Have a `LICENCE` file in the relevant sub folder:

   - https://github.com/tbarnes88/testSeleniumFrameworkCompare/blob/master/framework/Maven/apache-maven-3.2.3/LICENSE

4. Have a license file per dependency like `owin-LICENSE`

   - https://github.com/UniteIdeas/linksSDGs-Tagger/tree/master/solr-5.4.0/licenses

Comment (andreasohlund) [reply LICENSE.md:12]: Lets exclude the license for now and raise an issue in with your research?

Comment (andreasohlund) [reply LICENSE.md:12]: Disregard saw your other PR 👍

**Pull request closed without merging**